### PR TITLE
EN-42390: Make new config to optionally stop writes from Balboa to Cassandra

### PR DESCRIPTION
**Pull request by bnguyen12**
* New optional ENV variable to stop writes from Balboa (specifically `balboa-service-jms`) to Cassandra
* Typo fixes

### CODE REVIEW

File: balboa-service-jms/src/main/java/com/socrata/balboa/jms/activemq/Consumer.java [hunk 1]
```diff
@@ -37,28 +39,34 @@ public void onMessage(Message payload) {
         long start = System.currentTimeMillis();
         String messageText = null;
         try {
-            TextMessage text = (TextMessage) payload;
-            messageText = text.getText();
+            // Before we even consider chunking a large message, check if we even need to write to send out the message
+            if (stopWrites) {
+                session.commit();
+                log.debug("Config was set to stop writes from Balboa to Cassandra. Ignoring message");
```
Comment (bnguyen12) [inline balboa-service-jms/src/main/java/com/socrata/balboa/jms/activemq/Consumer.java:45]: This was the only thing added to this file. The git diff makes this look ugly tho

File: balboa-service-jms/src/main/java/com/socrata/balboa/jms/activemq/Consumer.java [hunk 2]
```diff
@@ -37,28 +39,34 @@ public void onMessage(Message payload) {
         long start = System.currentTimeMillis();
         String messageText = null;
         try {
-            TextMessage text = (TextMessage) payload;
-            messageText = text.getText();
+            // Before we consider chunking a large message, check if we even need to write to send out the message
+            if (stopWrites) {
+                session.commit();
```
Comment (bnguyen12) [inline balboa-service-jms/src/main/java/com/socrata/balboa/jms/activemq/Consumer.java:44]: is this even necessary? if no changes are being made to Cassandra then a commit isn't necessary? or are commits something you should regularly do, even during a NOOP?

Comment (progman32) [reply balboa-service-jms/src/main/java/com/socrata/balboa/jms/activemq/Consumer.java:44]: The session is related to ActiveMQ, not Cassandra. I believe we should commit here, else (afaik) the message won't be acknowledged. https://activemq.apache.org/how-do-transactions-work

**Review by progman32 — APPROVED** [review]

File: balboa-service-jms/src/main/java/com/socrata/balboa/jms/activemq/Consumer.java (hunk 2 above)
Comment (bnguyen12) [reply balboa-service-jms/src/main/java/com/socrata/balboa/jms/activemq/Consumer.java:44]: Ah ok good, I was wondering how messages were acked/nacked here

**Pull request merged**